CSS files/components/buttons/buttons.css

```diff
@@ -1,16 +1,18 @@
 
 .btn{
-    padding:0.8rem 1.2rem;
+    padding:8px 1rem;
```

Comment:
GIve both values in rem / px. Dont use two different units.

Reply:
Got your point. Just to clarify shouldn't we use pixels for values less than 10 pixels?